src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  let result = sourceString.split(';');
```

Comment:
The variable name 'result' is too generic and does not describe the contents of the variable. Consider using a more descriptive name, such as 'styleDeclarations' or 'cssRules', to improve code readability.